Project/DAQ_System/Program/Src/Sensor/GyroScope/LSM6DSOXGyroscope.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+* Gyroscope L3GD20H
```

Comment:
Comment is the wrong sensor

Reply:
Thanks for pointing this one out. I wouldn't have noticed this.